package.json

```diff
@@ -7,6 +7,7 @@
     "host": "localhost"
   },
   "scripts": {
+    "prepare": "cd devtools && bower install",
```

Comment:
I'm not sure that the bower dependency below corresponds to this, wouldn't it be `../node_modules/.bin/bower` ?

Reply:
It will, but `node_modules/bin` is automatically added to the path for npm scripts (at least I think that's what's happening and it's somewhat borne out with docs). 

This lets us remove Bower as a global package (and also means that we don't need it to be installed globally in Travis)

Reply:
> node_modules/bin is automatically added to the path for npm scripts

Ahh, that makes sense.